examples/azure/backup/build.sh

```diff
@@ -107,7 +107,7 @@ range     <= split($sequence, "\n")
 print("creating %q disks with size %q\n", $vm_disks_count, $vm_disks_size)
 
 for i in $range {
-	azure_vm_disk_attach_new($vm_name, $group, "disk" + $i, $vm_disks_size, "Premium_LRS")
+	azure_vm_disk_attach_new($vm_name, $group, "disk" + $i, $vm_disks_size, "Premium_LRS", "None")
 }
```

Comment:
Why none here? This will be the default configuration?

Reply:
No, it is just an example and I was not interested on showing different configurations for this on the example right now =P

Reply:
Understood :)

Reply:
But it would be cool to have examples for this too ;-)